scripts/common.py

```diff
@@ -325,7 +325,14 @@ def ensure_release_scripts_folder_exists_and_is_up_to_date():
                 'git@github.com:oppia/release-scripts.git'])
 
     with CD(release_scripts_dirpath):
+        ask_user_to_confirm(
+            'Please make sure that the release-scripts folder is clean and '
+            'you are on master branch in release-scripts folder.')
+        python_utils.PRINT('Verifying that release-scripts folder is clean...')
         verify_local_repo_is_clean()
+        python_utils.PRINT(
+            'Verifying that user is on master branch in '
+            'release-scripts folder...')
         verify_current_branch_name('master')
```

Comment:
Any specific reason for not doing this by the script and asking the coordinator to perform this? i.e, is there any issue with script switching to the master branch if required?

Reply:
If there are any changes on the branch user is currently on, that will create issues and they will need to manually interfere in that case.

Reply:
We are validating the branch to be clean on line 332, right? and if that passes we can change the branch to master (if required) right? OR I'm missing something?

Reply:
[I'm fine with the current flow as well!] 

Marking this thread resolved!